.gitlab-ci.yml

```diff
@@ -201,6 +201,33 @@ Regression:
       - "*.repo"
     when: always
 
+
+Image Tests:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+rhel-9.0-[^ga][\S]+/ && $NIGHTLY == "true" && $RHEL_MAJOR == "9"'
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER =~ /[\S]+rhel-8.6-[^ga][\S]+/ && $NIGHTLY == "true" && $RHEL_MAJOR == "8"'
+  script:
+    - schutzbot/deploy.sh
+    - /usr/libexec/tests/osbuild-composer/image_tests.sh
+  interruptible: true
```

Comment:
IDK, I've copied the job definition. @jrusz can explain this in more details b/c he's added the `interruptible` glag in 8136e0f06

From the commit log it looks like I need to remove this flag? @jrusz is my understanding correct ?

Reply:
@atodorov I don't think this matters TBH, it's `false` by default, if you set it to true, it will just probably cancel this job if you run the pipeline again while this was still running. I'd leave it as is.